README.md

```diff
@@ -1,6 +1,6 @@
 # Code Climate PHP_CodeSniffer Engine
 
-`codeclimate-phpcodesniffer` is a Code Climate engine that wraps the [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) static analysis tool. You can run it on your command line using the Code Climate CLI, or on our hosted analysis platform.
+`codeclimate-phpcodesniffer` is a Code Climate engine that wraps the [PHP_Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) static analysis tool. You can run it on your command line using the Code Climate CLI, or on our hosted analysis platform.
```

Comment:
nit: They seem to stylize their own name as `PHP_CodeSniffer`, so if we're going to use the underscore we should probably drop the space.